core/src/com/unciv/ui/newgamescreen/PlayerPickerTable.kt

```diff
@@ -260,12 +275,12 @@ class PlayerPickerTable(val previousScreen: IPreviousScreen, var gameParameters:
     private fun getAvailablePlayerCivs(): ArrayList<Nation> {
         var nations = ArrayList<Nation>()
         for (nation in previousScreen.gameSetupInfo.ruleset.nations.values
-                .filter { it.isMajorCiv() }) {
-            if (gameParameters.players.any { it.chosenCiv == nation.name })
+                .filter { it.isMajorCiv() || it.isSpectator() }) {
+            // skip already chosen civs, except for spectator
+            if (gameParameters.players.any { it.chosenCiv == nation.name && it.chosenCiv != "Spectator"})
```

Comment:
??? 
Didn't you say that only one spectator is available?
I'm confused

Reply:
In the future I would like to have a general possibility to have more than one spectator. That is why in Player table we can add multiple spectators. However there is a temporary limit on 1 spectator when pressing rightSideButton right now.